main.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+window.addEventListener('load', main);
+
+function main() {
+  setBallPosition();
+}
+
+function setBallPosition() {
```

Comment:
It is not a `setBallPosition`, this name will be good for function inside listener(line 30), but not for this

You can just remove this function and move all code to main, or rename it